### ISSUE TITLE
[FR] Add Balancer

### ISSUE THREAD

**Issue by yant0** [GittyMac/WalletCount#8]
Add feature of dividing the budget per day for that month to know how much you can spend 👍

**Comment by ZineeEddine** [comment]
Hi, @yant0 

I think I could work on this feature please give me more details.

The app is great, I like it

do you think something like in the picture.

I tried to make a program that reads the value of the budget for example 102.00$, and it tells you that it makes per week 26$ (which is rounded Math.round normally it is 25.5$)
and also per day.

You can also make it appear in the widget.

What do you think about this program?

<img width="250" alt="Capture d’écran 2023-02-14 à 19 14 10" src="https://user-images.githubusercontent.com/92603736/218824129-ea8244fc-239e-4ae7-9d3f-0cc8745ef244.png"><img width="250" alt="Capture d’écran 2023-02-14 à 19 29 23" src="https://user-images.githubusercontent.com/92603736/218826322-e68a07b0-30a1-4348-b6a0-2562ab345bb0.png">

**Comment by ZineeEddine** [comment]
Hi, @GittyMac 

Is the project still in development?

@yant0, I tried to work on your functionality, but I got no response.

**Comment by GittyMac** [comment]
> Hi, @GittyMac 
> 
> Is the project still in development?

Yes, WalletCount is still in development. I'm working on a big update that should fix most of the issues. Currently I'm focusing on adding an expenses list. I'm however only working on this on and off, so progress is slow.

Hopefully later on I'll be able to focus on this issue.

**Comment by ZineeEddine** [comment]
Hi, @GittyMac, 
yes, I understand, it's completely normal (congratulations already for the project).

I wanted to work on this feature #8, but I didn't get an answer from the member, if you like the way I display the data, I can just correct the decimal conversion, and why not do a PR , for the new feature Add Balancer, would be nice to have this feature. The application itself is already very good.
